client/src/components/InterpreterSignUpForm.js

```diff
@@ -32,8 +55,11 @@ class InterpreterSignUpForm extends Component {
       languages
     } = this.state;
     // perform all neccassary validations
-    if (password !== confirmationPassword) {
-      alert("Passwords don't match");
+    if (0 === name.length) {
```

Comment:
Comments above can be removed. Focus more on having a clear code with good naming convention rather than writing comments :)